modules/api-explorer-client/src/components/OperationResult/Response.tsx

```diff
@@ -1,35 +1,33 @@
-import React from 'react'
-import { Segment, Tab, Message } from 'semantic-ui-react'
-import JSONTree from 'react-json-tree'
+import React from "react";
+import { Segment, Tab, Message } from "semantic-ui-react/index";
+import JSONTree from "react-json-tree";
 
 type Props = {
-  loading: boolean,
-  expanded: boolean,
-  response: any,
-  error: ?Error,
-}
+  loading: boolean;
+  expanded: boolean;
+  response: any;
+  error: Error | null | undefined;
```

Comment:
it can be written as 
```
error?: Error
```
with `?` on the key

Reply:
I think the `error? : Error` and `error: ?Error` is different
https://stackoverflow.com/questions/51182335/flow-type-question-mark

Reply:
`error? : Error`  implies that error may **not** be provided, thus it will be either `Error` or `undefined`

`error: ?Error` implies that error will **always** be provided, but the value can be `undefined` or `null` 

In this case, error will not always be provided, so I think `error?: Error` is more appropriate.

Reply:
so, do you mean that the original type is not appropriate?
got it, fixed it